master.css

```diff
@@ -36,6 +48,9 @@ td .anchor {
 td:first-child {
     min-width: 320px;
 }
+td, th {
+    border: 1px solid #fff;
```

Comment:
This seems to cause the scroll performance in FF... I changed the border-collapse to collapse and added this one because otherwise the scrolling rows would appear below the sticky header...